superdesk/publish/formatters/ninjs_ftp_formatter.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2014 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+
+from .ninjs_formatter import NINJS2Formatter
+from superdesk.media.renditions import get_rendition_file_name
+
+
+class FTPNinjsFormatter(NINJS2Formatter):
```

Comment:
We will need this in Superdesk v1.31, so it might be better to inherit from the original NINJSFormatter